### ISSUE TITLE
Issue: Can't add core skill that isn't predefined

### ISSUE THREAD

**Issue by Matthew-Burfield** [FCC-Alumni/alumni-network#163]
There's no option to select "other" or type in my own skill.

Specifically, I wanted to add Webpack as a skill, but couldn't

**Comment by bonham000** [comment]
I knew this was going to be brought up... :face_with_head_bandage:  I'll let @no-stack-dub-sack handle this one!

**Comment by no-stack-dub-sack** [comment]
@Matthew-Burfield @bonham000 I just tried to look for it, but must have deleted it at some point... but this was on a TODO list in one of our components forever! I also knew this would come up. Pretty easy to implement, I think this one is def doable.

**Comment by Matthew-Burfield** [comment]
Some other things to consider are: if someone adds a new skill that isn't on the pre-defined list, will that skill still be searchable? Would it still link to a wikipedia page? Would new skills become available in the predefined list?

**Comment by no-stack-dub-sack** [comment]
@Matthew-Burfield Yes, if we went with this option, the new item would essentially become part of the permanent data for the app and it would be searchable just like any other skill and link to wiki, etc.

However, I did implement a solution for this in PR #167, but it seemed to raise more questions than it did solve problems. I am going to try one more approach that will keep everything out of the backend that may work better, but there still may be some issues around it. If that ends up being the case, I think a more well-rounded list will end up being the solution for now - we should be able to cover 99% of the most important skills people would want to add, and anything left would be really obscure. 

Even if my new approach does work, the main issue I see is how to prevent people from spamming the skills and interests with random things that are not coding related? I've seen packages that prevent curse words and the like, but can't think of anything that would prevent someone from saying that "plumbing" is a coding interest of theirs...
